airflow/www_rbac/api/experimental/endpoints.py

```diff
@@ -281,15 +305,25 @@ def cancel_dag_run(dag_id):
     run_id = data['run_id']
 
     try:
-       dr = _cancel_dag_run(dag_id, run_id)
+        with create_session() as session:
+            dr = models.DagRun.find(dag_id=dag_id, run_id=run_id)[0]
+            session.add(models.Log(
+                'api_cancel_dagrun',
+                task_instance=None,
+                owner=_get_user_from_header(),
```

Comment:
Is the owner the username of someone from the User table?

Reply:
yes